lib/datadog/tracing/contrib/http.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require_relative 'component'
+require_relative 'http/integration'
+require_relative 'http/distributed/propagation'
+
+module Datadog
+  module Tracing
+    module Contrib
+      # `Net/HTTP` integration public API
+      module HTTP
+        # Inject distributed headers into the given request
+        # @param digest [Datadog::Tracing::TraceDigest] the trace to inject
+        # @param data [Hash] the request to inject
+        def self.inject(digest, data)
+          raise 'Please invoke Datadog.configure at least once before calling this method' unless @propagation
```

Comment:
I'm experimenting with raising an actionable error, instead of possibly lazy loading all of `ddtrace` inside this method.